source/resampling_with_code2.Rmd

```diff
@@ -466,29 +478,21 @@ z
 
 Of course we could just keep doing this: run the {{< var cell >}} corresponding
 to a trial, above, to get a new count, and then store it at the next position
-in the `z` {{< var array_name >}} with:
+in the `z` {{< var array >}} with:
 [`z[1] = count`]{.python}[`z[2] <- count`]{.r},
 and so on, but that also looks like a task that the computer is well-suited
-for.  And indeed it is, because {{< var lang >}} an excellent way of repeating
-the same set of steps many times — the `for` loop.  You have already seen a
+for.  And indeed it is, because {{< var lang >}} has an excellent way of
+repeating the same set of steps many times — the `for` loop.  You have already seen a
 preview of the `for` loop in @sec-resampling-method. Here we dive into them in
 more depth.
 
-## For-loops in {{< var lang >}}
-
-::: todo
-Need to switch for-loops to use integers rather than strings here.
+:::{.callout-box}
+### For-loops in {{< var lang >}}
 
-Can we get away without "print"?  Probably not, but then, need to introduce
-it.
-:::
+<!-- Matthew wants to switch this to integers instead of strings; S needs to find out why -->
```

Comment:
Because we haven't introduced strings yet ...

Reply:
I think strings are quite intuitive here; we could mention them earlier if you feel they need explicit introduction.

Reply:
I think it's fine if we introduce them properly, but strings are a significant thing.  At the moment we just have numbers.  So I think we'd have to work out where we've got the space to put the introduction to strings.

Reply:
Where do we note these TODOs?

Reply:
How about `notes/todo.md`?